src/styles/blocks/card-stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2:nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3:nth-child(-n + 3) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The `:nth-child` selector is used incorrectly here. The `:nth-child` selector applies styles to the children of an element, but it seems you want to apply styles to the first few stars based on the rating. Consider using a different approach, such as applying a class to the stars that should be active.